### PR TITLE
Remove @compiled/cli package

### PR DESCRIPTION
**Pull request by JakeLane**
As we've now enrolled in the [codeshift community](https://www.codeshiftcommunity.com/docs/) codemod command, we can remove the cli package if we update the docs to reference the new command.

### CODE REVIEW

File: packages/codemods/README.md [hunk 1]
```diff
@@ -2,10 +2,23 @@
 
 > Codemods for easy migration from [styled components](https://styled-components.com/) and [emotion](https://emotion.sh/docs/introduction).
 
+## Usage
```
Comment (pancaspe87) [inline packages/codemods/README.md:5]: Should we also update/remove https://compiledcssinjs.com/docs/pkg-cli ?

File: scripts/generate-renovate-changeset.ts [hunk 2]
```diff
@@ -1,23 +0,0 @@
-// This script is designed so that Renovate can automatically add changesets to the PRs it
-// creates. It is designed to be run in the root of the repo.
```
Comment (pancaspe87) [inline scripts/generate-renovate-changeset.ts:2]: Why are we removing this?

**Review by liamqma — APPROVED** [review]

File: packages/codemods/README.md (hunk 1 above)
Comment (JakeLane) [reply packages/codemods/README.md:5]: yep will do after merging this PR

File: scripts/generate-renovate-changeset.ts (hunk 2 above)
Comment (JakeLane) [reply scripts/generate-renovate-changeset.ts:2]: Unfortunately it doesn't work anymore due to internal security compliance. We'll need to come up with a different solution

Comment (pancaspe87) [reply scripts/generate-renovate-changeset.ts:2]: so not strictly related to this PR, but something to do anyway.

**Review by pancaspe87 — APPROVED** [review]

**Pull request merged**